WcaOnRails/app/models/team_member.rb

```diff
@@ -11,6 +11,9 @@ class TeamMember < ApplicationRecord
   scope :current_leader, -> { self.current.merge(self.leader) }
 
   attr_accessor :current_user
+  delegate :friendly_id, to: :team
+  delegate :hidden?, to: :team
```

Comment:
Yes, because it's related to another slight cleanup I did: we use to do the following to include `teams`:
```ruby
      teams: current_team_members.includes(:team).reject do |team_member|
        team_member.team.hidden?
      end.map do |team_member|
        {
          friendly_id: team_member.team.friendly_id,
          leader: team_member.team_leader?,
        }
```

By delegating `hidden?` to the team and providing a default serializable hash, it's now equivalent to this:
```ruby
  json[:teams] = current_team_members.includes(:team).reject(&:hidden?)
```